app/templates/play/level/modal/hero-victory-modal.jade

```diff
@@ -90,6 +91,8 @@ block modal-body-content
             for item in rewards.items
               - var item = thangTypes[item];
               .reward-panel.item(data-item-thang-type=item.get('original'))
+                if item.get('subscriber')
+                  .subscribe-text(data-i18n="play.subscribers_only") Subscriber Only!
```

Comment:
English text should not be included in the template. It hides erroneous i18n strings, and can get out of sync with what's in en.coffee.